parquet-common/src/main/java/org/apache/parquet/bytes/MultiBufferInputStream.java

```diff
@@ -248,13 +295,17 @@ public int read() throws IOException {
       throw new EOFException();
     }
 
+    this.position++;
```

Comment:
Can you revert the unnecessary change here? This was `this.position += 1;` and I don't see a reason to change it more than necessary.